### PR TITLE
docs: Created documentation page for Hoppie ACARS System

### LINKED ISSUE #421: Create a guide for CPDLC
The guide should cover how to get connected on the Hoppie network and how you can use this in the A32NX. The relevant PR is here:
https://github.com/flybywiresim/a32nx/pull/6451

I think the testing instructions on that PR cover a fair bit. For the rest, we can talk sven - who did the implementation - directly.

### PR DESCRIPTION
**Pull request by frankkopp**
Closes #421

## Summary
Feature Guide for Hoppie ACARS System

- [x] Draft
- [x] Expand
- [x] New Screenshots and imageoptimization
- [x] Review

### Location
Feature Guides

Discord username (if different from GitHub): Cdr_Maverick#6475

### CODE REVIEW

**Comment by Valastiri** [comment]
check build

File: docs/fbw-a32nx/feature-guides/hoppie.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    - Datalink Status
+    - Automatic D-ATIS, DCL, OCL
+    - Vert, Lat requests, Text
+    - Other requests, when can we expect
+    - Emergency
+    - Automatic preparation of replies for report-requests
+    - FPL changes for Lat requests
+    - Flight Monitoring for conditional reports
+
+    This plan might change at any time.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/hoppie.md]: Maybe state first what features "are currently available" and then what is planned after?

File: docs/fbw-a32nx/feature-guides/hoppie.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+The Hoppie system is developed by Jeroen "Hoppie" Hoppenbrouwers which provides text-based communication between aircraft, virtual airlines and virtual ATC stations. It provides multiple communication protocols to communicate between different stations. The system is used by controllers on VATSIM and IVAO.
+
+[Hoppie Homepage](http://www.hoppie.nl/pub/){target=new}
+
+The A32NX implements the TELEX and CPDLC communication between ATC stations and the aircraft.
+
+!!! note "[CPDLC](https://skybrary.aero/articles/controller-pilot-data-link-communications-cpdlc){target=new} stands for Controller Pilot Data Link Communication."
+
+It allows a communication from ATC stations with the aircraft without the use of voice, but it is still required that the pilot monitors the voice frequency.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/hoppie.md]:
```suggestion
It allows communication between ATC stations and the aircraft without the use of voice, but it is still required that the pilot monitors the voice frequency.
```

File: docs/fbw-a32nx/feature-guides/hoppie.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
+# Hoppie ACARS
+
+!!! warning "Not available in the Stable Version"
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/hoppie.md:3]:
```suggestion
!!! warning "Not available in the Stable Version"

!!! warning "Requires use and credentials with Hoppie service"
```

Maybe include this at the top feel free to adjust the admon as you see fit.

File: docs/fbw-a32nx/feature-guides/hoppie.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+## Troubleshooting
+
+Sometimes the ATC station does not respond to PDC or CPDLC messages. This mostly happens because of ATC controllers not responding, an issue in the Hoppie network or potentially an issue with the A32NX.
+
+This chapter will give you some hints to check what the actual cause of the issue is.
+
+### Find online stations
+
+It is possible to find all currently online stations. This is helpful to check if the responsible ATC station provides CPDLC or PDC.
+A complete list can be found [here](http://www.hoppie.nl/acars/system/online.html).
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/hoppie.md]:
```suggestion
A complete list can be found [here](http://www.hoppie.nl/acars/system/online.html){target=new}.
```

**Review by Valastiri — CHANGES_REQUESTED** [review]
Suggestions + Changes

**Comment by Valastiri** [comment]
Reference: https://github.com/flybywiresim/a32nx/pull/6451

File: docs/fbw-a32nx/feature-guides/hoppie.md (hunk 1 above)
Comment (frankkopp) [reply docs/fbw-a32nx/feature-guides/hoppie.md]: will do

**Review by Valastiri — APPROVED** [review]

**Pull request merged**